scripts/install-linux-deps.sh

```diff
@@ -54,6 +46,18 @@ env DEBIAN_FRONTEND=noninteractive apt-get install --yes --quiet \
     unzip \
     zlib1g-dev \
   && rm -rf /var/lib/apt/lists/*
+  
+# build prime_server from source
+readonly primeserver_version=0.7.0
+readonly primeserver_dir=/usr/local/src/prime_server
+git clone --recurse-submodules https://github.com/kevinkreiser/prime_server $primeserver_dir
+pushd $primeserver_dir
+git checkout $primeserver_version && \
+cmake -B build . && \
+cmake --build build -- -j${CONCURRENCY:-$(nproc)} && \
+make -C build install && \
+popd && \
+rm -r $primeserver_dir
```

Comment:
btw, autotools build didn't work out-of-the-box, same old https://github.com/kevinkreiser/prime_server/issues/100

Reply:
I guess Ubuntu finally caught up with Mac & Arch from 2 years back haha

Reply:
I didn't write the cmake stuff and don't quite trust it. can you switch to autotools build for the time being? once we thirdparty it into cmake we will have to straighten it out

Reply:
it has to work I'll check it out

Reply:
In the ubuntu 22.04 container with only the `install_linux_deps.sh` script ran, it failed to build the image with that exact error on my machine

Reply:
i just did it. you have to install all the deps, following the readme line by line:

```
kkreiser@X570-I-AORUS-PRO-WIFI:~/sandbox/valhalla$ docker run -it ubuntu:22.04 /bin/bash
root@1d504fa4b33b:/# apt update
Get:1 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy InRelease [270 kB]
Get:3 http://security.ubuntu.com/ubuntu jammy-security/multiverse amd64 Packages [4732 B]
Get:4 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [729 kB]
Get:5 http://security.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [670 kB]
Get:6 http://security.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [786 kB] 
Get:7 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [114 kB]        
Get:8 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [99.8 kB]
23% [Waiting for headers]      
Get:9 http://archive.ubuntu.com/ubuntu jammy/restricted amd64 Packages [164 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy/multiverse amd64 Packages [266 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages [17.5 MB]
Get:12 http://archive.ubuntu.com/ubuntu jammy/main amd64 Packages [1792 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [990 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy-updates/multiverse amd64 Packages [8978 B]
Get:15 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [718 kB]
Get:16 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [1043 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy-backports/universe amd64 Packages [8738 B]
Get:18 http://archive.ubuntu.com/ubuntu jammy-backports/main amd64 Packages [3520 B]
Fetched 25.3 MB in 4s (6367 kB/s)                          
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
root@1d504fa4b33b:/# apt install git
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  ca-certificates git-man less libbrotli1 libbsd0 libcbor0.8 libcurl3-gnutls libedit2 liberror-perl libexpat1 libfido2-1
  libgdbm-compat4 libgdbm6 libldap-2.5-0 libldap-common libmd0 libnghttp2-14 libperl5.34 libpsl5 librtmp1 libsasl2-2 libsasl2-modules
  libsasl2-modules-db libssh-4 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxmuu1 netbase openssh-client openssl patch
  perl perl-modules-5.34 publicsuffix xauth
Suggested packages:
  gettext-base git-daemon-run | git-daemon-sysvinit git-doc git-email git-gui gitk gitweb git-cvs git-mediawiki git-svn gdbm-l10n
  libsasl2-modules-gssapi-mit | libsasl2-modules-gssapi-heimdal libsasl2-modules-ldap libsasl2-modules-otp libsasl2-modules-sql
  keychain libpam-ssh monkeysphere ssh-askpass ed diffutils-doc perl-doc libterm-readline-gnu-perl | libterm-readline-perl-perl make
  libtap-harness-archive-perl
The following NEW packages will be installed:
  ca-certificates git git-man less libbrotli1 libbsd0 libcbor0.8 libcurl3-gnutls libedit2 liberror-perl libexpat1 libfido2-1
  libgdbm-compat4 libgdbm6 libldap-2.5-0 libldap-common libmd0 libnghttp2-14 libperl5.34 libpsl5 librtmp1 libsasl2-2 libsasl2-modules
  libsasl2-modules-db libssh-4 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxmuu1 netbase openssh-client openssl patch
  perl perl-modules-5.34 publicsuffix xauth
0 upgraded, 40 newly installed, 0 to remove and 0 not upgraded.
Need to get 17.4 MB of archives.
After this operation, 84.0 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Abort.
root@1d504fa4b33b:/# apt install git
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  ca-certificates git-man less libbrotli1 libbsd0 libcbor0.8 libcurl3-gnutls libedit2 liberror-perl libexpat1 libfido2-1
  libgdbm-compat4 libgdbm6 libldap-2.5-0 libldap-common libmd0 libnghttp2-14 libperl5.34 libpsl5 librtmp1 libsasl2-2 libsasl2-modules
  libsasl2-modules-db libssh-4 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxmuu1 netbase openssh-client openssl patch
  perl perl-modules-5.34 publicsuffix xauth
Suggested packages:
  gettext-base git-daemon-run | git-daemon-sysvinit git-doc git-email git-gui gitk gitweb git-cvs git-mediawiki git-svn gdbm-l10n
  libsasl2-modules-gssapi-mit | libsasl2-modules-gssapi-heimdal libsasl2-modules-ldap libsasl2-modules-otp libsasl2-modules-sql
  keychain libpam-ssh monkeysphere ssh-askpass ed diffutils-doc perl-doc libterm-readline-gnu-perl | libterm-readline-perl-perl make
  libtap-harness-archive-perl
The following NEW packages will be installed:
  ca-certificates git git-man less libbrotli1 libbsd0 libcbor0.8 libcurl3-gnutls libedit2 liberror-perl libexpat1 libfido2-1
  libgdbm-compat4 libgdbm6 libldap-2.5-0 libldap-common libmd0 libnghttp2-14 libperl5.34 libpsl5 librtmp1 libsasl2-2 libsasl2-modules
  libsasl2-modules-db libssh-4 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxmuu1 netbase openssh-client openssl patch
  perl perl-modules-5.34 publicsuffix xauth
0 upgraded, 40 newly installed, 0 to remove and 0 not upgraded.
Need to get 17.4 MB of archives.
After this operation, 84.0 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 perl-modules-5.34 all 5.34.0-3ubuntu1.1 [2976 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 libgdbm6 amd64 1.23-1 [33.9 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy/main amd64 libgdbm-compat4 amd64 1.23-1 [6606 B]
Get:4 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libperl5.34 amd64 5.34.0-3ubuntu1.1 [4819 kB]
Get:5 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 perl amd64 5.34.0-3ubuntu1.1 [232 kB]
Get:6 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openssl amd64 3.0.2-0ubuntu1.7 [1183 kB]
Get:7 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 ca-certificates all 20211016ubuntu0.22.04.1 [144 kB]
Get:8 http://archive.ubuntu.com/ubuntu jammy/main amd64 less amd64 590-1build1 [143 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy/main amd64 libmd0 amd64 1.0.4-1build1 [23.0 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy/main amd64 libbsd0 amd64 0.11.5-1 [44.8 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libexpat1 amd64 2.4.7-1ubuntu0.2 [91.0 kB]
Get:12 http://archive.ubuntu.com/ubuntu jammy/main amd64 netbase all 6.3 [12.9 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy/main amd64 libcbor0.8 amd64 0.8.0-2ubuntu1 [24.6 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy/main amd64 libedit2 amd64 3.1-20210910-1build1 [96.8 kB]
Get:15 http://archive.ubuntu.com/ubuntu jammy/main amd64 libfido2-1 amd64 1.10.0-1 [82.8 kB]
Get:16 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnghttp2-14 amd64 1.43.0-1build3 [76.3 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy/main amd64 libpsl5 amd64 0.21.0-1.2build2 [58.4 kB]
Get:18 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxau6 amd64 1:1.0.9-1build5 [7634 B]
Get:19 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxdmcp6 amd64 1:1.1.3-0ubuntu5 [10.9 kB]
Get:20 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxcb1 amd64 1.14-3ubuntu3 [49.0 kB]
Get:21 http://archive.ubuntu.com/ubuntu jammy/main amd64 libx11-data all 2:1.7.5-1 [119 kB]
Get:22 http://archive.ubuntu.com/ubuntu jammy/main amd64 libx11-6 amd64 2:1.7.5-1 [666 kB]
Get:23 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxext6 amd64 2:1.3.4-1build1 [31.8 kB]
Get:24 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxmuu1 amd64 2:1.1.3-3 [10.2 kB]
Get:25 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openssh-client amd64 1:8.9p1-3ubuntu0.1 [908 kB]
Get:26 http://archive.ubuntu.com/ubuntu jammy/main amd64 publicsuffix all 20211207.1025-1 [129 kB]
Get:27 http://archive.ubuntu.com/ubuntu jammy/main amd64 xauth amd64 1:1.1-1build2 [27.5 kB]
Get:28 http://archive.ubuntu.com/ubuntu jammy/main amd64 libbrotli1 amd64 1.0.9-2build6 [315 kB]
Get:29 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libsasl2-modules-db amd64 2.1.27+dfsg2-3ubuntu1.1 [20.6 kB]
Get:30 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libsasl2-2 amd64 2.1.27+dfsg2-3ubuntu1.1 [53.8 kB]
Get:31 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libldap-2.5-0 amd64 2.5.13+dfsg-0ubuntu0.22.04.1 [183 kB]
Get:32 http://archive.ubuntu.com/ubuntu jammy/main amd64 librtmp1 amd64 2.4+20151223.gitfa8646d.1-2build4 [58.2 kB]
Get:33 http://archive.ubuntu.com/ubuntu jammy/main amd64 libssh-4 amd64 0.9.6-2build1 [184 kB]
Get:34 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libcurl3-gnutls amd64 7.81.0-1ubuntu1.7 [283 kB]
Get:35 http://archive.ubuntu.com/ubuntu jammy/main amd64 liberror-perl all 0.17029-1 [26.5 kB]
Get:36 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 git-man all 1:2.34.1-1ubuntu1.5 [953 kB]
Get:37 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 git amd64 1:2.34.1-1ubuntu1.5 [3132 kB]
Get:38 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libldap-common all 2.5.13+dfsg-0ubuntu0.22.04.1 [15.9 kB]
Get:39 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libsasl2-modules amd64 2.1.27+dfsg2-3ubuntu1.1 [57.2 kB]
Get:40 http://archive.ubuntu.com/ubuntu jammy/main amd64 patch amd64 2.7.6-7build2 [109 kB]
Fetched 17.4 MB in 2s (7636 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package perl-modules-5.34.
(Reading database ... 4395 files and directories currently installed.)
Preparing to unpack .../00-perl-modules-5.34_5.34.0-3ubuntu1.1_all.deb ...
Unpacking perl-modules-5.34 (5.34.0-3ubuntu1.1) ...
Selecting previously unselected package libgdbm6:amd64.
Preparing to unpack .../01-libgdbm6_1.23-1_amd64.deb ...
Unpacking libgdbm6:amd64 (1.23-1) ...
Selecting previously unselected package libgdbm-compat4:amd64.
Preparing to unpack .../02-libgdbm-compat4_1.23-1_amd64.deb ...
Unpacking libgdbm-compat4:amd64 (1.23-1) ...
Selecting previously unselected package libperl5.34:amd64.
Preparing to unpack .../03-libperl5.34_5.34.0-3ubuntu1.1_amd64.deb ...
Unpacking libperl5.34:amd64 (5.34.0-3ubuntu1.1) ...
Selecting previously unselected package perl.
Preparing to unpack .../04-perl_5.34.0-3ubuntu1.1_amd64.deb ...
Unpacking perl (5.34.0-3ubuntu1.1) ...
Selecting previously unselected package openssl.
Preparing to unpack .../05-openssl_3.0.2-0ubuntu1.7_amd64.deb ...
Unpacking openssl (3.0.2-0ubuntu1.7) ...
Selecting previously unselected package ca-certificates.
Preparing to unpack .../06-ca-certificates_20211016ubuntu0.22.04.1_all.deb ...
Unpacking ca-certificates (20211016ubuntu0.22.04.1) ...
Selecting previously unselected package less.
Preparing to unpack .../07-less_590-1build1_amd64.deb ...
Unpacking less (590-1build1) ...
Selecting previously unselected package libmd0:amd64.
Preparing to unpack .../08-libmd0_1.0.4-1build1_amd64.deb ...
Unpacking libmd0:amd64 (1.0.4-1build1) ...
Selecting previously unselected package libbsd0:amd64.
Preparing to unpack .../09-libbsd0_0.11.5-1_amd64.deb ...
Unpacking libbsd0:amd64 (0.11.5-1) ...
Selecting previously unselected package libexpat1:amd64.
Preparing to unpack .../10-libexpat1_2.4.7-1ubuntu0.2_amd64.deb ...
Unpacking libexpat1:amd64 (2.4.7-1ubuntu0.2) ...
Selecting previously unselected package netbase.
Preparing to unpack .../11-netbase_6.3_all.deb ...
Unpacking netbase (6.3) ...
Selecting previously unselected package libcbor0.8:amd64.
Preparing to unpack .../12-libcbor0.8_0.8.0-2ubuntu1_amd64.deb ...
Unpacking libcbor0.8:amd64 (0.8.0-2ubuntu1) ...
Selecting previously unselected package libedit2:amd64.
Preparing to unpack .../13-libedit2_3.1-20210910-1build1_amd64.deb ...
Unpacking libedit2:amd64 (3.1-20210910-1build1) ...
Selecting previously unselected package libfido2-1:amd64.
Preparing to unpack .../14-libfido2-1_1.10.0-1_amd64.deb ...
Unpacking libfido2-1:amd64 (1.10.0-1) ...
Selecting previously unselected package libnghttp2-14:amd64.
Preparing to unpack .../15-libnghttp2-14_1.43.0-1build3_amd64.deb ...
Unpacking libnghttp2-14:amd64 (1.43.0-1build3) ...
Selecting previously unselected package libpsl5:amd64.
Preparing to unpack .../16-libpsl5_0.21.0-1.2build2_amd64.deb ...
Unpacking libpsl5:amd64 (0.21.0-1.2build2) ...
Selecting previously unselected package libxau6:amd64.
Preparing to unpack .../17-libxau6_1%3a1.0.9-1build5_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.9-1build5) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../18-libxdmcp6_1%3a1.1.3-0ubuntu5_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.3-0ubuntu5) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../19-libxcb1_1.14-3ubuntu3_amd64.deb ...
Unpacking libxcb1:amd64 (1.14-3ubuntu3) ...
Selecting previously unselected package libx11-data.
Preparing to unpack .../20-libx11-data_2%3a1.7.5-1_all.deb ...
Unpacking libx11-data (2:1.7.5-1) ...
Selecting previously unselected package libx11-6:amd64.
Preparing to unpack .../21-libx11-6_2%3a1.7.5-1_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.7.5-1) ...
Selecting previously unselected package libxext6:amd64.
Preparing to unpack .../22-libxext6_2%3a1.3.4-1build1_amd64.deb ...
Unpacking libxext6:amd64 (2:1.3.4-1build1) ...
Selecting previously unselected package libxmuu1:amd64.
Preparing to unpack .../23-libxmuu1_2%3a1.1.3-3_amd64.deb ...
Unpacking libxmuu1:amd64 (2:1.1.3-3) ...
Selecting previously unselected package openssh-client.
Preparing to unpack .../24-openssh-client_1%3a8.9p1-3ubuntu0.1_amd64.deb ...
Unpacking openssh-client (1:8.9p1-3ubuntu0.1) ...
Selecting previously unselected package publicsuffix.
Preparing to unpack .../25-publicsuffix_20211207.1025-1_all.deb ...
Unpacking publicsuffix (20211207.1025-1) ...
Selecting previously unselected package xauth.
Preparing to unpack .../26-xauth_1%3a1.1-1build2_amd64.deb ...
Unpacking xauth (1:1.1-1build2) ...
Selecting previously unselected package libbrotli1:amd64.
Preparing to unpack .../27-libbrotli1_1.0.9-2build6_amd64.deb ...
Unpacking libbrotli1:amd64 (1.0.9-2build6) ...
Selecting previously unselected package libsasl2-modules-db:amd64.
Preparing to unpack .../28-libsasl2-modules-db_2.1.27+dfsg2-3ubuntu1.1_amd64.deb ...
Unpacking libsasl2-modules-db:amd64 (2.1.27+dfsg2-3ubuntu1.1) ...
Selecting previously unselected package libsasl2-2:amd64.
Preparing to unpack .../29-libsasl2-2_2.1.27+dfsg2-3ubuntu1.1_amd64.deb ...
Unpacking libsasl2-2:amd64 (2.1.27+dfsg2-3ubuntu1.1) ...
Selecting previously unselected package libldap-2.5-0:amd64.
Preparing to unpack .../30-libldap-2.5-0_2.5.13+dfsg-0ubuntu0.22.04.1_amd64.deb ...
Unpacking libldap-2.5-0:amd64 (2.5.13+dfsg-0ubuntu0.22.04.1) ...
Selecting previously unselected package librtmp1:amd64.
Preparing to unpack .../31-librtmp1_2.4+20151223.gitfa8646d.1-2build4_amd64.deb ...
Unpacking librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2build4) ...
Selecting previously unselected package libssh-4:amd64.
Preparing to unpack .../32-libssh-4_0.9.6-2build1_amd64.deb ...
Unpacking libssh-4:amd64 (0.9.6-2build1) ...
Selecting previously unselected package libcurl3-gnutls:amd64.
Preparing to unpack .../33-libcurl3-gnutls_7.81.0-1ubuntu1.7_amd64.deb ...
Unpacking libcurl3-gnutls:amd64 (7.81.0-1ubuntu1.7) ...
Selecting previously unselected package liberror-perl.
Preparing to unpack .../34-liberror-perl_0.17029-1_all.deb ...
Unpacking liberror-perl (0.17029-1) ...
Selecting previously unselected package git-man.
Preparing to unpack .../35-git-man_1%3a2.34.1-1ubuntu1.5_all.deb ...
Unpacking git-man (1:2.34.1-1ubuntu1.5) ...
Selecting previously unselected package git.
Preparing to unpack .../36-git_1%3a2.34.1-1ubuntu1.5_amd64.deb ...
Unpacking git (1:2.34.1-1ubuntu1.5) ...
Selecting previously unselected package libldap-common.
Preparing to unpack .../37-libldap-common_2.5.13+dfsg-0ubuntu0.22.04.1_all.deb ...
Unpacking libldap-common (2.5.13+dfsg-0ubuntu0.22.04.1) ...
Selecting previously unselected package libsasl2-modules:amd64.
Preparing to unpack .../38-libsasl2-modules_2.1.27+dfsg2-3ubuntu1.1_amd64.deb ...
Unpacking libsasl2-modules:amd64 (2.1.27+dfsg2-3ubuntu1.1) ...
Selecting previously unselected package patch.
Preparing to unpack .../39-patch_2.7.6-7build2_amd64.deb ...
Unpacking patch (2.7.6-7build2) ...
Setting up libexpat1:amd64 (2.4.7-1ubuntu0.2) ...
Setting up libxau6:amd64 (1:1.0.9-1build5) ...
Setting up libpsl5:amd64 (0.21.0-1.2build2) ...
Setting up libcbor0.8:amd64 (0.8.0-2ubuntu1) ...
Setting up libbrotli1:amd64 (1.0.9-2build6) ...
Setting up libsasl2-modules:amd64 (2.1.27+dfsg2-3ubuntu1.1) ...
Setting up libnghttp2-14:amd64 (1.43.0-1build3) ...
Setting up less (590-1build1) ...
Setting up perl-modules-5.34 (5.34.0-3ubuntu1.1) ...
Setting up libldap-common (2.5.13+dfsg-0ubuntu0.22.04.1) ...
Setting up libsasl2-modules-db:amd64 (2.1.27+dfsg2-3ubuntu1.1) ...
Setting up libx11-data (2:1.7.5-1) ...
Setting up librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2build4) ...
Setting up patch (2.7.6-7build2) ...
Setting up libsasl2-2:amd64 (2.1.27+dfsg2-3ubuntu1.1) ...
Setting up libssh-4:amd64 (0.9.6-2build1) ...
Setting up libmd0:amd64 (1.0.4-1build1) ...
Setting up git-man (1:2.34.1-1ubuntu1.5) ...
Setting up netbase (6.3) ...
Setting up libfido2-1:amd64 (1.10.0-1) ...
Setting up openssl (3.0.2-0ubuntu1.7) ...
Setting up libbsd0:amd64 (0.11.5-1) ...
Setting up publicsuffix (20211207.1025-1) ...
Setting up libgdbm6:amd64 (1.23-1) ...
Setting up libxdmcp6:amd64 (1:1.1.3-0ubuntu5) ...
Setting up libxcb1:amd64 (1.14-3ubuntu3) ...
Setting up libedit2:amd64 (3.1-20210910-1build1) ...
Setting up libldap-2.5-0:amd64 (2.5.13+dfsg-0ubuntu0.22.04.1) ...
Setting up ca-certificates (20211016ubuntu0.22.04.1) ...
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/
Dialog.pm line 78.)
debconf: falling back to frontend: Readline
Updating certificates in /etc/ssl/certs...
124 added, 0 removed; done.
Setting up libgdbm-compat4:amd64 (1.23-1) ...
Setting up libx11-6:amd64 (2:1.7.5-1) ...
Setting up libxmuu1:amd64 (2:1.1.3-3) ...
Setting up libperl5.34:amd64 (5.34.0-3ubuntu1.1) ...
Setting up openssh-client (1:8.9p1-3ubuntu0.1) ...
update-alternatives: using /usr/bin/ssh to provide /usr/bin/rsh (rsh) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/rsh.1.gz because associated file /usr/share/man/man1/ssh.1.gz (of lin
k group rsh) doesn't exist
update-alternatives: using /usr/bin/slogin to provide /usr/bin/rlogin (rlogin) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/rlogin.1.gz because associated file /usr/share/man/man1/slogin.1.gz (
of link group rlogin) doesn't exist
update-alternatives: using /usr/bin/scp to provide /usr/bin/rcp (rcp) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/rcp.1.gz because associated file /usr/share/man/man1/scp.1.gz (of lin
k group rcp) doesn't exist
Setting up libxext6:amd64 (2:1.3.4-1build1) ...
Setting up libcurl3-gnutls:amd64 (7.81.0-1ubuntu1.7) ...
Setting up perl (5.34.0-3ubuntu1.1) ...
Setting up xauth (1:1.1-1build2) ...
Setting up liberror-perl (0.17029-1) ...
Setting up git (1:2.34.1-1ubuntu1.5) ...
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...
Processing triggers for ca-certificates (20211016ubuntu0.22.04.1) ...
Updating certificates in /etc/ssl/certs...
0 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d...
done.
root@1d504fa4b33b:/# git clone https://github.com/kevinkreiser/prime_server
Cloning into 'prime_server'...
remote: Enumerating objects: 1701, done.
remote: Counting objects: 100% (198/198), done.
remote: Compressing objects: 100% (118/118), done.
remote: Total 1701 (delta 103), reused 146 (delta 68), pack-reused 1503
Receiving objects: 100% (1701/1701), 570.90 KiB | 2.06 MiB/s, done.
Resolving deltas: 100% (1117/1117), done.
root@1d504fa4b33b:/# cd prime_server/
root@1d504fa4b33b:/prime_server# less README
root@1d504fa4b33b:/prime_server# less README
root@1d504fa4b33b:/prime_server# sudo apt-get install autoconf automake pkg-config libtool make gcc g++ lcov
bash: sudo: command not found
root@1d504fa4b33b:/prime_server# apt-get install autoconf automake pkg-config libtool make gcc g++ lcov
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  autotools-dev binutils binutils-common binutils-x86-64-linux-gnu bzip2 cpp cpp-11 file fontconfig-config fonts-dejavu-core g++-11
  gcc-11 gcc-11-base libasan6 libatomic1 libbinutils libc-dev-bin libc-devtools libc6-dev libcc1-0 libcommon-sense-perl libcrypt-dev
  libctf-nobfd0 libctf0 libdeflate0 libdpkg-perl libfile-fcntllock-perl libfontconfig1 libfreetype6 libgcc-11-dev libgd-perl libgd3
  libglib2.0-0 libglib2.0-data libgomp1 libicu70 libisl23 libitm1 libjbig0 libjpeg-turbo8 libjpeg8 libjson-perl libjson-xs-perl
  liblocale-gettext-perl liblsan0 libltdl-dev libltdl7 libmagic-mgc libmagic1 libmpc3 libmpfr6 libnsl-dev libperlio-gzip-perl
  libpng16-16 libquadmath0 libsigsegv2 libstdc++-11-dev libtiff5 libtirpc-dev libtsan0 libtypes-serialiser-perl libubsan1 libwebp7
  libxml2 libxpm4 linux-libc-dev m4 manpages manpages-dev rpcsvc-proto shared-mime-info ucf xdg-user-dirs xz-utils
Suggested packages:
  autoconf-archive gnu-standards autoconf-doc gettext binutils-doc bzip2-doc cpp-doc gcc-11-locales g++-multilib g++-11-multilib
  gcc-11-doc gcc-multilib flex bison gdb gcc-doc gcc-11-multilib glibc-doc debian-keyring gnupg bzr libgd-tools libtool-doc
  libstdc++-11-doc gfortran | fortran95-compiler gcj-jdk m4-doc make-doc man-browser dpkg-dev
The following NEW packages will be installed:
  autoconf automake autotools-dev binutils binutils-common binutils-x86-64-linux-gnu bzip2 cpp cpp-11 file fontconfig-config
  fonts-dejavu-core g++ g++-11 gcc gcc-11 gcc-11-base lcov libasan6 libatomic1 libbinutils libc-dev-bin libc-devtools libc6-dev
  libcc1-0 libcommon-sense-perl libcrypt-dev libctf-nobfd0 libctf0 libdeflate0 libdpkg-perl libfile-fcntllock-perl libfontconfig1
  libfreetype6 libgcc-11-dev libgd-perl libgd3 libglib2.0-0 libglib2.0-data libgomp1 libicu70 libisl23 libitm1 libjbig0 libjpeg-turbo8
  libjpeg8 libjson-perl libjson-xs-perl liblocale-gettext-perl liblsan0 libltdl-dev libltdl7 libmagic-mgc libmagic1 libmpc3 libmpfr6
  libnsl-dev libperlio-gzip-perl libpng16-16 libquadmath0 libsigsegv2 libstdc++-11-dev libtiff5 libtirpc-dev libtool libtsan0
  libtypes-serialiser-perl libubsan1 libwebp7 libxml2 libxpm4 linux-libc-dev m4 make manpages manpages-dev pkg-config rpcsvc-proto
  shared-mime-info ucf xdg-user-dirs xz-utils
0 upgraded, 82 newly installed, 0 to remove and 0 not upgraded.
Need to get 85.3 MB of archives.
After this operation, 287 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://archive.ubuntu.com/ubuntu jammy/main amd64 liblocale-gettext-perl amd64 1.07-4build3 [17.1 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libglib2.0-0 amd64 2.72.4-0ubuntu1 [1461 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libglib2.0-data all 2.72.4-0ubuntu1 [4882 B]
Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 libicu70 amd64 70.1-2 [10.6 MB]
Get:5 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libxml2 amd64 2.9.13+dfsg-1ubuntu0.2 [764 kB]
Get:6 http://archive.ubuntu.com/ubuntu jammy/main amd64 shared-mime-info amd64 2.1-2 [454 kB]
Get:7 http://archive.ubuntu.com/ubuntu jammy/main amd64 ucf all 3.0043 [56.1 kB]
Get:8 http://archive.ubuntu.com/ubuntu jammy/main amd64 xdg-user-dirs amd64 0.17-2ubuntu4 [53.9 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy/main amd64 libmagic-mgc amd64 1:5.41-3 [257 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy/main amd64 libmagic1 amd64 1:5.41-3 [87.2 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy/main amd64 file amd64 1:5.41-3 [21.5 kB]
Get:12 http://archive.ubuntu.com/ubuntu jammy/main amd64 libpng16-16 amd64 1.6.37-3build5 [191 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy/main amd64 manpages all 5.10-1ubuntu1 [1375 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy/main amd64 xz-utils amd64 5.2.5-2ubuntu1 [84.8 kB]
Get:15 http://archive.ubuntu.com/ubuntu jammy/main amd64 libsigsegv2 amd64 2.13-1ubuntu3 [14.6 kB]
Get:16 http://archive.ubuntu.com/ubuntu jammy/main amd64 m4 amd64 1.4.18-5ubuntu2 [199 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy/main amd64 autoconf all 2.71-2 [338 kB]
Get:18 http://archive.ubuntu.com/ubuntu jammy/main amd64 autotools-dev all 20220109.1 [44.9 kB]
Get:19 http://archive.ubuntu.com/ubuntu jammy/main amd64 automake all 1:1.16.5-1.3 [558 kB]
Get:20 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 binutils-common amd64 2.38-4ubuntu2.1 [221 kB]
Get:21 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libbinutils amd64 2.38-4ubuntu2.1 [661 kB]
Get:22 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libctf-nobfd0 amd64 2.38-4ubuntu2.1 [107 kB]
Get:23 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libctf0 amd64 2.38-4ubuntu2.1 [103 kB]
Get:24 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 binutils-x86-64-linux-gnu amd64 2.38-4ubuntu2.1 [2328 kB]
Get:25 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 binutils amd64 2.38-4ubuntu2.1 [3198 B]
Get:26 http://archive.ubuntu.com/ubuntu jammy/main amd64 bzip2 amd64 1.0.8-5build1 [34.8 kB]
Get:27 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 gcc-11-base amd64 11.3.0-1ubuntu1~22.04 [20.8 kB]
Get:28 http://archive.ubuntu.com/ubuntu jammy/main amd64 libisl23 amd64 0.24-2build1 [727 kB]
Get:29 http://archive.ubuntu.com/ubuntu jammy/main amd64 libmpfr6 amd64 4.1.0-3build3 [1425 kB]
Get:30 http://archive.ubuntu.com/ubuntu jammy/main amd64 libmpc3 amd64 1.2.1-2build1 [46.9 kB]
Get:31 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 cpp-11 amd64 11.3.0-1ubuntu1~22.04 [9967 kB]
Get:32 http://archive.ubuntu.com/ubuntu jammy/main amd64 cpp amd64 4:11.2.0-1ubuntu1 [27.7 kB]
Get:33 http://archive.ubuntu.com/ubuntu jammy/main amd64 fonts-dejavu-core all 2.37-2build1 [1041 kB]
Get:34 http://archive.ubuntu.com/ubuntu jammy/main amd64 fontconfig-config all 2.13.1-4.2ubuntu5 [29.1 kB]
Get:35 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libcc1-0 amd64 12.1.0-2ubuntu1~22.04 [47.4 kB]
Get:36 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libgomp1 amd64 12.1.0-2ubuntu1~22.04 [126 kB]
Get:37 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libitm1 amd64 12.1.0-2ubuntu1~22.04 [30.2 kB]
Get:38 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libatomic1 amd64 12.1.0-2ubuntu1~22.04 [10.4 kB]
Get:39 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libasan6 amd64 11.3.0-1ubuntu1~22.04 [2284 kB]
Get:40 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 liblsan0 amd64 12.1.0-2ubuntu1~22.04 [1069 kB]
Get:41 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libtsan0 amd64 11.3.0-1ubuntu1~22.04 [2262 kB]
Get:42 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libubsan1 amd64 12.1.0-2ubuntu1~22.04 [976 kB]
Get:43 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libquadmath0 amd64 12.1.0-2ubuntu1~22.04 [154 kB]
Get:44 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libgcc-11-dev amd64 11.3.0-1ubuntu1~22.04 [2517 kB]
Get:45 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 gcc-11 amd64 11.3.0-1ubuntu1~22.04 [20.1 MB]
Get:46 http://archive.ubuntu.com/ubuntu jammy/main amd64 gcc amd64 4:11.2.0-1ubuntu1 [5112 B]                                          
Get:47 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libc-dev-bin amd64 2.35-0ubuntu3.1 [20.4 kB]                          
Get:48 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 linux-libc-dev amd64 5.15.0-58.64 [1347 kB]                           
Get:49 http://archive.ubuntu.com/ubuntu jammy/main amd64 libcrypt-dev amd64 1:4.4.27-1 [112 kB]                                        
Get:50 http://archive.ubuntu.com/ubuntu jammy/main amd64 rpcsvc-proto amd64 1.4.2-0ubuntu6 [68.5 kB]                                   
Get:51 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libtirpc-dev amd64 1.3.2-2ubuntu0.1 [192 kB]                          
Get:52 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnsl-dev amd64 1.3.0-2build2 [71.3 kB]                                      
Get:53 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libc6-dev amd64 2.35-0ubuntu3.1 [2099 kB]                             
Get:54 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libstdc++-11-dev amd64 11.3.0-1ubuntu1~22.04 [2087 kB]                
Get:55 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 g++-11 amd64 11.3.0-1ubuntu1~22.04 [11.4 MB]                          
Get:56 http://archive.ubuntu.com/ubuntu jammy/main amd64 g++ amd64 4:11.2.0-1ubuntu1 [1412 B]                                          
Get:57 http://archive.ubuntu.com/ubuntu jammy/main amd64 libjson-perl all 4.04000-1 [81.8 kB]                                          
Get:58 http://archive.ubuntu.com/ubuntu jammy/main amd64 libperlio-gzip-perl amd64 0.19-1build8 [14.9 kB]                              
Get:59 http://archive.ubuntu.com/ubuntu jammy/universe amd64 lcov all 1.15-1 [99.5 kB]                                                 
Get:60 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libfreetype6 amd64 2.11.1+dfsg-1ubuntu0.1 [389 kB]                    
Get:61 http://archive.ubuntu.com/ubuntu jammy/main amd64 libfontconfig1 amd64 2.13.1-4.2ubuntu5 [131 kB]                               
Get:62 http://archive.ubuntu.com/ubuntu jammy/main amd64 libjpeg-turbo8 amd64 2.1.2-0ubuntu1 [134 kB]                                  
Get:63 http://archive.ubuntu.com/ubuntu jammy/main amd64 libjpeg8 amd64 8c-2ubuntu10 [2264 B]                                          
Get:64 http://archive.ubuntu.com/ubuntu jammy/main amd64 libdeflate0 amd64 1.10-2 [70.9 kB]                                            
Get:65 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libjbig0 amd64 2.1-3.1ubuntu0.22.04.1 [29.2 kB]                       
Get:66 http://archive.ubuntu.com/ubuntu jammy/main amd64 libwebp7 amd64 1.2.2-2 [206 kB]                                               
Get:67 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libtiff5 amd64 4.3.0-6ubuntu0.3 [183 kB]                              
Get:68 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxpm4 amd64 1:3.5.12-1build2 [36.2 kB]                                      
Get:69 http://archive.ubuntu.com/ubuntu jammy/main amd64 libgd3 amd64 2.3.0-2ubuntu2 [129 kB]                                          
Get:70 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libc-devtools amd64 2.35-0ubuntu3.1 [28.9 kB]                         
Get:71 http://archive.ubuntu.com/ubuntu jammy/main amd64 libcommon-sense-perl amd64 3.75-2build1 [21.1 kB]                             
Get:72 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libdpkg-perl all 1.21.1ubuntu2.1 [237 kB]                             
Get:73 http://archive.ubuntu.com/ubuntu jammy/main amd64 libfile-fcntllock-perl amd64 0.22-3build7 [33.9 kB]                           
Get:74 http://archive.ubuntu.com/ubuntu jammy/main amd64 libtypes-serialiser-perl all 1.01-1 [11.6 kB]                                 
Get:75 http://archive.ubuntu.com/ubuntu jammy/main amd64 libjson-xs-perl amd64 4.030-1build3 [87.2 kB]                                 
Get:76 http://archive.ubuntu.com/ubuntu jammy/main amd64 libltdl7 amd64 2.4.6-15build2 [39.6 kB]                                       
Get:77 http://archive.ubuntu.com/ubuntu jammy/main amd64 libltdl-dev amd64 2.4.6-15build2 [169 kB]                                     
Get:78 http://archive.ubuntu.com/ubuntu jammy/main amd64 libtool all 2.4.6-15build2 [164 kB]                                           
Get:79 http://archive.ubuntu.com/ubuntu jammy/main amd64 make amd64 4.3-4.1build1 [180 kB]                                             
Get:80 http://archive.ubuntu.com/ubuntu jammy/main amd64 manpages-dev all 5.10-1ubuntu1 [2309 kB]                                      
Get:81 http://archive.ubuntu.com/ubuntu jammy/main amd64 pkg-config amd64 0.29.2-1ubuntu3 [48.2 kB]                                    
Get:82 http://archive.ubuntu.com/ubuntu jammy/main amd64 libgd-perl amd64 2.76-2build1 [138 kB]                                        
Fetched 85.3 MB in 10s (8517 kB/s)                                                                                                     
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package liblocale-gettext-perl.
(Reading database ... 8349 files and directories currently installed.)
Preparing to unpack .../00-liblocale-gettext-perl_1.07-4build3_amd64.deb ...
Unpacking liblocale-gettext-perl (1.07-4build3) ...
Selecting previously unselected package libglib2.0-0:amd64.
Preparing to unpack .../01-libglib2.0-0_2.72.4-0ubuntu1_amd64.deb ...
Unpacking libglib2.0-0:amd64 (2.72.4-0ubuntu1) ...
Selecting previously unselected package libglib2.0-data.
Preparing to unpack .../02-libglib2.0-data_2.72.4-0ubuntu1_all.deb ...
Unpacking libglib2.0-data (2.72.4-0ubuntu1) ...
Selecting previously unselected package libicu70:amd64.
Preparing to unpack .../03-libicu70_70.1-2_amd64.deb ...
Unpacking libicu70:amd64 (70.1-2) ...
Selecting previously unselected package libxml2:amd64.
Preparing to unpack .../04-libxml2_2.9.13+dfsg-1ubuntu0.2_amd64.deb ...
Unpacking libxml2:amd64 (2.9.13+dfsg-1ubuntu0.2) ...
Selecting previously unselected package shared-mime-info.
Preparing to unpack .../05-shared-mime-info_2.1-2_amd64.deb ...
Unpacking shared-mime-info (2.1-2) ...
Selecting previously unselected package ucf.
Preparing to unpack .../06-ucf_3.0043_all.deb ...
Moving old data out of the way
Unpacking ucf (3.0043) ...
Selecting previously unselected package xdg-user-dirs.
Preparing to unpack .../07-xdg-user-dirs_0.17-2ubuntu4_amd64.deb ...
Unpacking xdg-user-dirs (0.17-2ubuntu4) ...
Selecting previously unselected package libmagic-mgc.
Preparing to unpack .../08-libmagic-mgc_1%3a5.41-3_amd64.deb ...
Unpacking libmagic-mgc (1:5.41-3) ...
Selecting previously unselected package libmagic1:amd64.
Preparing to unpack .../09-libmagic1_1%3a5.41-3_amd64.deb ...
Unpacking libmagic1:amd64 (1:5.41-3) ...
Selecting previously unselected package file.
Preparing to unpack .../10-file_1%3a5.41-3_amd64.deb ...
Unpacking file (1:5.41-3) ...
Selecting previously unselected package libpng16-16:amd64.
Preparing to unpack .../11-libpng16-16_1.6.37-3build5_amd64.deb ...
Unpacking libpng16-16:amd64 (1.6.37-3build5) ...
Selecting previously unselected package manpages.
Preparing to unpack .../12-manpages_5.10-1ubuntu1_all.deb ...
Unpacking manpages (5.10-1ubuntu1) ...
Selecting previously unselected package xz-utils.
Preparing to unpack .../13-xz-utils_5.2.5-2ubuntu1_amd64.deb ...
Unpacking xz-utils (5.2.5-2ubuntu1) ...
Selecting previously unselected package libsigsegv2:amd64.
Preparing to unpack .../14-libsigsegv2_2.13-1ubuntu3_amd64.deb ...
Unpacking libsigsegv2:amd64 (2.13-1ubuntu3) ...
Selecting previously unselected package m4.
Preparing to unpack .../15-m4_1.4.18-5ubuntu2_amd64.deb ...
Unpacking m4 (1.4.18-5ubuntu2) ...
Selecting previously unselected package autoconf.
Preparing to unpack .../16-autoconf_2.71-2_all.deb ...
Unpacking autoconf (2.71-2) ...
Selecting previously unselected package autotools-dev.
Preparing to unpack .../17-autotools-dev_20220109.1_all.deb ...
Unpacking autotools-dev (20220109.1) ...
Selecting previously unselected package automake.
Preparing to unpack .../18-automake_1%3a1.16.5-1.3_all.deb ...
Unpacking automake (1:1.16.5-1.3) ...
Selecting previously unselected package binutils-common:amd64.
Preparing to unpack .../19-binutils-common_2.38-4ubuntu2.1_amd64.deb ...
Unpacking binutils-common:amd64 (2.38-4ubuntu2.1) ...
Selecting previously unselected package libbinutils:amd64.
Preparing to unpack .../20-libbinutils_2.38-4ubuntu2.1_amd64.deb ...
Unpacking libbinutils:amd64 (2.38-4ubuntu2.1) ...
Selecting previously unselected package libctf-nobfd0:amd64.
Preparing to unpack .../21-libctf-nobfd0_2.38-4ubuntu2.1_amd64.deb ...
Unpacking libctf-nobfd0:amd64 (2.38-4ubuntu2.1) ...
Selecting previously unselected package libctf0:amd64.
Preparing to unpack .../22-libctf0_2.38-4ubuntu2.1_amd64.deb ...
Unpacking libctf0:amd64 (2.38-4ubuntu2.1) ...
Selecting previously unselected package binutils-x86-64-linux-gnu.
Preparing to unpack .../23-binutils-x86-64-linux-gnu_2.38-4ubuntu2.1_amd64.deb ...
Unpacking binutils-x86-64-linux-gnu (2.38-4ubuntu2.1) ...
Selecting previously unselected package binutils.
Preparing to unpack .../24-binutils_2.38-4ubuntu2.1_amd64.deb ...
Unpacking binutils (2.38-4ubuntu2.1) ...
Selecting previously unselected package bzip2.
Preparing to unpack .../25-bzip2_1.0.8-5build1_amd64.deb ...
Unpacking bzip2 (1.0.8-5build1) ...
Selecting previously unselected package gcc-11-base:amd64.
Preparing to unpack .../26-gcc-11-base_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking gcc-11-base:amd64 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package libisl23:amd64.
Preparing to unpack .../27-libisl23_0.24-2build1_amd64.deb ...
Unpacking libisl23:amd64 (0.24-2build1) ...
Selecting previously unselected package libmpfr6:amd64.
Preparing to unpack .../28-libmpfr6_4.1.0-3build3_amd64.deb ...
Unpacking libmpfr6:amd64 (4.1.0-3build3) ...
Selecting previously unselected package libmpc3:amd64.
Preparing to unpack .../29-libmpc3_1.2.1-2build1_amd64.deb ...
Unpacking libmpc3:amd64 (1.2.1-2build1) ...
Selecting previously unselected package cpp-11.
Preparing to unpack .../30-cpp-11_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking cpp-11 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package cpp.
Preparing to unpack .../31-cpp_4%3a11.2.0-1ubuntu1_amd64.deb ...
Unpacking cpp (4:11.2.0-1ubuntu1) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../32-fonts-dejavu-core_2.37-2build1_all.deb ...
Unpacking fonts-dejavu-core (2.37-2build1) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../33-fontconfig-config_2.13.1-4.2ubuntu5_all.deb ...
Unpacking fontconfig-config (2.13.1-4.2ubuntu5) ...
Selecting previously unselected package libcc1-0:amd64.
Preparing to unpack .../34-libcc1-0_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking libcc1-0:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libgomp1:amd64.
Preparing to unpack .../35-libgomp1_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking libgomp1:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libitm1:amd64.
Preparing to unpack .../36-libitm1_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking libitm1:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libatomic1:amd64.
Preparing to unpack .../37-libatomic1_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking libatomic1:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libasan6:amd64.
Preparing to unpack .../38-libasan6_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking libasan6:amd64 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package liblsan0:amd64.
Preparing to unpack .../39-liblsan0_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking liblsan0:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libtsan0:amd64.
Preparing to unpack .../40-libtsan0_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking libtsan0:amd64 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package libubsan1:amd64.
Preparing to unpack .../41-libubsan1_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking libubsan1:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libquadmath0:amd64.
Preparing to unpack .../42-libquadmath0_12.1.0-2ubuntu1~22.04_amd64.deb ...
Unpacking libquadmath0:amd64 (12.1.0-2ubuntu1~22.04) ...
Selecting previously unselected package libgcc-11-dev:amd64.
Preparing to unpack .../43-libgcc-11-dev_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking libgcc-11-dev:amd64 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package gcc-11.
Preparing to unpack .../44-gcc-11_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking gcc-11 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package gcc.
Preparing to unpack .../45-gcc_4%3a11.2.0-1ubuntu1_amd64.deb ...
Unpacking gcc (4:11.2.0-1ubuntu1) ...
Selecting previously unselected package libc-dev-bin.
Preparing to unpack .../46-libc-dev-bin_2.35-0ubuntu3.1_amd64.deb ...
Unpacking libc-dev-bin (2.35-0ubuntu3.1) ...
Selecting previously unselected package linux-libc-dev:amd64.
Preparing to unpack .../47-linux-libc-dev_5.15.0-58.64_amd64.deb ...
Unpacking linux-libc-dev:amd64 (5.15.0-58.64) ...
Selecting previously unselected package libcrypt-dev:amd64.
Preparing to unpack .../48-libcrypt-dev_1%3a4.4.27-1_amd64.deb ...
Unpacking libcrypt-dev:amd64 (1:4.4.27-1) ...
Selecting previously unselected package rpcsvc-proto.
Preparing to unpack .../49-rpcsvc-proto_1.4.2-0ubuntu6_amd64.deb ...
Unpacking rpcsvc-proto (1.4.2-0ubuntu6) ...
Selecting previously unselected package libtirpc-dev:amd64.
Preparing to unpack .../50-libtirpc-dev_1.3.2-2ubuntu0.1_amd64.deb ...
Unpacking libtirpc-dev:amd64 (1.3.2-2ubuntu0.1) ...
Selecting previously unselected package libnsl-dev:amd64.
Preparing to unpack .../51-libnsl-dev_1.3.0-2build2_amd64.deb ...
Unpacking libnsl-dev:amd64 (1.3.0-2build2) ...
Selecting previously unselected package libc6-dev:amd64.
Preparing to unpack .../52-libc6-dev_2.35-0ubuntu3.1_amd64.deb ...
Unpacking libc6-dev:amd64 (2.35-0ubuntu3.1) ...
Selecting previously unselected package libstdc++-11-dev:amd64.
Preparing to unpack .../53-libstdc++-11-dev_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking libstdc++-11-dev:amd64 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package g++-11.
Preparing to unpack .../54-g++-11_11.3.0-1ubuntu1~22.04_amd64.deb ...
Unpacking g++-11 (11.3.0-1ubuntu1~22.04) ...
Selecting previously unselected package g++.
Preparing to unpack .../55-g++_4%3a11.2.0-1ubuntu1_amd64.deb ...
Unpacking g++ (4:11.2.0-1ubuntu1) ...
Selecting previously unselected package libjson-perl.
Preparing to unpack .../56-libjson-perl_4.04000-1_all.deb ...
Unpacking libjson-perl (4.04000-1) ...
Selecting previously unselected package libperlio-gzip-perl.
Preparing to unpack .../57-libperlio-gzip-perl_0.19-1build8_amd64.deb ...
Unpacking libperlio-gzip-perl (0.19-1build8) ...
Selecting previously unselected package lcov.
Preparing to unpack .../58-lcov_1.15-1_all.deb ...
Unpacking lcov (1.15-1) ...
Selecting previously unselected package libfreetype6:amd64.
Preparing to unpack .../59-libfreetype6_2.11.1+dfsg-1ubuntu0.1_amd64.deb ...
Unpacking libfreetype6:amd64 (2.11.1+dfsg-1ubuntu0.1) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../60-libfontconfig1_2.13.1-4.2ubuntu5_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.13.1-4.2ubuntu5) ...
Selecting previously unselected package libjpeg-turbo8:amd64.
Preparing to unpack .../61-libjpeg-turbo8_2.1.2-0ubuntu1_amd64.deb ...
Unpacking libjpeg-turbo8:amd64 (2.1.2-0ubuntu1) ...
Selecting previously unselected package libjpeg8:amd64.
Preparing to unpack .../62-libjpeg8_8c-2ubuntu10_amd64.deb ...
Unpacking libjpeg8:amd64 (8c-2ubuntu10) ...
Selecting previously unselected package libdeflate0:amd64.
Preparing to unpack .../63-libdeflate0_1.10-2_amd64.deb ...
Unpacking libdeflate0:amd64 (1.10-2) ...
Selecting previously unselected package libjbig0:amd64.
Preparing to unpack .../64-libjbig0_2.1-3.1ubuntu0.22.04.1_amd64.deb ...
Unpacking libjbig0:amd64 (2.1-3.1ubuntu0.22.04.1) ...
Selecting previously unselected package libwebp7:amd64.
Preparing to unpack .../65-libwebp7_1.2.2-2_amd64.deb ...
Unpacking libwebp7:amd64 (1.2.2-2) ...
Selecting previously unselected package libtiff5:amd64.
Preparing to unpack .../66-libtiff5_4.3.0-6ubuntu0.3_amd64.deb ...
Unpacking libtiff5:amd64 (4.3.0-6ubuntu0.3) ...
Selecting previously unselected package libxpm4:amd64.
Preparing to unpack .../67-libxpm4_1%3a3.5.12-1build2_amd64.deb ...
Unpacking libxpm4:amd64 (1:3.5.12-1build2) ...
Selecting previously unselected package libgd3:amd64.
Preparing to unpack .../68-libgd3_2.3.0-2ubuntu2_amd64.deb ...
Unpacking libgd3:amd64 (2.3.0-2ubuntu2) ...
Selecting previously unselected package libc-devtools.
Preparing to unpack .../69-libc-devtools_2.35-0ubuntu3.1_amd64.deb ...
Unpacking libc-devtools (2.35-0ubuntu3.1) ...
Selecting previously unselected package libcommon-sense-perl:amd64.
Preparing to unpack .../70-libcommon-sense-perl_3.75-2build1_amd64.deb ...
Unpacking libcommon-sense-perl:amd64 (3.75-2build1) ...
Selecting previously unselected package libdpkg-perl.
Preparing to unpack .../71-libdpkg-perl_1.21.1ubuntu2.1_all.deb ...
Unpacking libdpkg-perl (1.21.1ubuntu2.1) ...
Selecting previously unselected package libfile-fcntllock-perl.
Preparing to unpack .../72-libfile-fcntllock-perl_0.22-3build7_amd64.deb ...
Unpacking libfile-fcntllock-perl (0.22-3build7) ...
Selecting previously unselected package libtypes-serialiser-perl.
Preparing to unpack .../73-libtypes-serialiser-perl_1.01-1_all.deb ...
Unpacking libtypes-serialiser-perl (1.01-1) ...
Selecting previously unselected package libjson-xs-perl.
Preparing to unpack .../74-libjson-xs-perl_4.030-1build3_amd64.deb ...
Unpacking libjson-xs-perl (4.030-1build3) ...
Selecting previously unselected package libltdl7:amd64.
Preparing to unpack .../75-libltdl7_2.4.6-15build2_amd64.deb ...
Unpacking libltdl7:amd64 (2.4.6-15build2) ...
Selecting previously unselected package libltdl-dev:amd64.
Preparing to unpack .../76-libltdl-dev_2.4.6-15build2_amd64.deb ...
Unpacking libltdl-dev:amd64 (2.4.6-15build2) ...
Selecting previously unselected package libtool.
Preparing to unpack .../77-libtool_2.4.6-15build2_all.deb ...
Unpacking libtool (2.4.6-15build2) ...
Selecting previously unselected package make.
Preparing to unpack .../78-make_4.3-4.1build1_amd64.deb ...
Unpacking make (4.3-4.1build1) ...
Selecting previously unselected package manpages-dev.
Preparing to unpack .../79-manpages-dev_5.10-1ubuntu1_all.deb ...
Unpacking manpages-dev (5.10-1ubuntu1) ...
Selecting previously unselected package pkg-config.
Preparing to unpack .../80-pkg-config_0.29.2-1ubuntu3_amd64.deb ...
Unpacking pkg-config (0.29.2-1ubuntu3) ...
Selecting previously unselected package libgd-perl.
Preparing to unpack .../81-libgd-perl_2.76-2build1_amd64.deb ...
Unpacking libgd-perl (2.76-2build1) ...
Setting up gcc-11-base:amd64 (11.3.0-1ubuntu1~22.04) ...
Setting up libxpm4:amd64 (1:3.5.12-1build2) ...
Setting up xdg-user-dirs (0.17-2ubuntu4) ...
Setting up libmagic-mgc (1:5.41-3) ...
Setting up libfile-fcntllock-perl (0.22-3build7) ...
Setting up libglib2.0-0:amd64 (2.72.4-0ubuntu1) ...
No schema files found: doing nothing.
Setting up manpages (5.10-1ubuntu1) ...
Setting up binutils-common:amd64 (2.38-4ubuntu2.1) ...
Setting up libmagic1:amd64 (1:5.41-3) ...
Setting up libdeflate0:amd64 (1.10-2) ...
Setting up libcommon-sense-perl:amd64 (3.75-2build1) ...
Setting up linux-libc-dev:amd64 (5.15.0-58.64) ...
Setting up libctf-nobfd0:amd64 (2.38-4ubuntu2.1) ...
Setting up file (1:5.41-3) ...
Setting up libgomp1:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up bzip2 (1.0.8-5build1) ...
Setting up libjbig0:amd64 (2.1-3.1ubuntu0.22.04.1) ...
Setting up libasan6:amd64 (11.3.0-1ubuntu1~22.04) ...
Setting up autotools-dev (20220109.1) ...
Setting up libglib2.0-data (2.72.4-0ubuntu1) ...
Setting up libtirpc-dev:amd64 (1.3.2-2ubuntu0.1) ...
Setting up rpcsvc-proto (1.4.2-0ubuntu6) ...
Setting up make (4.3-4.1build1) ...
Setting up libmpfr6:amd64 (4.1.0-3build3) ...
Setting up libsigsegv2:amd64 (2.13-1ubuntu3) ...
Setting up xz-utils (5.2.5-2ubuntu1) ...
update-alternatives: using /usr/bin/xz to provide /usr/bin/lzma (lzma) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/lzma.1.gz because associated file /usr/share/man/man1/xz.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/unlzma.1.gz because associated file /usr/share/man/man1/unxz.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzcat.1.gz because associated file /usr/share/man/man1/xzcat.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzmore.1.gz because associated file /usr/share/man/man1/xzmore.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzless.1.gz because associated file /usr/share/man/man1/xzless.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzdiff.1.gz because associated file /usr/share/man/man1/xzdiff.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzcmp.1.gz because associated file /usr/share/man/man1/xzcmp.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzgrep.1.gz because associated file /usr/share/man/man1/xzgrep.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzegrep.1.gz because associated file /usr/share/man/man1/xzegrep.1.gz (of link group lzma) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/lzfgrep.1.gz because associated file /usr/share/man/man1/xzfgrep.1.gz (of link group lzma) doesn't exist
Setting up libquadmath0:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up libpng16-16:amd64 (1.6.37-3build5) ...
Setting up libmpc3:amd64 (1.2.1-2build1) ...
Setting up libatomic1:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up fonts-dejavu-core (2.37-2build1) ...
Setting up ucf (3.0043) ...
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78.)
debconf: falling back to frontend: Readline
Setting up libjpeg-turbo8:amd64 (2.1.2-0ubuntu1) ...
Setting up libltdl7:amd64 (2.4.6-15build2) ...
Setting up libdpkg-perl (1.21.1ubuntu2.1) ...
Setting up libwebp7:amd64 (1.2.2-2) ...
Setting up libubsan1:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up libtypes-serialiser-perl (1.01-1) ...
Setting up libnsl-dev:amd64 (1.3.0-2build2) ...
Setting up libcrypt-dev:amd64 (1:4.4.27-1) ...
Setting up libjson-perl (4.04000-1) ...
Setting up libbinutils:amd64 (2.38-4ubuntu2.1) ...
Setting up libisl23:amd64 (0.24-2build1) ...
Setting up libc-dev-bin (2.35-0ubuntu3.1) ...
Setting up libperlio-gzip-perl (0.19-1build8) ...
Setting up libcc1-0:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up liblocale-gettext-perl (1.07-4build3) ...
Setting up liblsan0:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up libitm1:amd64 (12.1.0-2ubuntu1~22.04) ...
Setting up libicu70:amd64 (70.1-2) ...
Setting up libtsan0:amd64 (11.3.0-1ubuntu1~22.04) ...
Setting up libctf0:amd64 (2.38-4ubuntu2.1) ...
Setting up libjpeg8:amd64 (8c-2ubuntu10) ...
Setting up libjson-xs-perl (4.030-1build3) ...
Setting up cpp-11 (11.3.0-1ubuntu1~22.04) ...
Setting up manpages-dev (5.10-1ubuntu1) ...
Setting up fontconfig-config (2.13.1-4.2ubuntu5) ...
Setting up m4 (1.4.18-5ubuntu2) ...
Setting up libfreetype6:amd64 (2.11.1+dfsg-1ubuntu0.1) ...
Setting up pkg-config (0.29.2-1ubuntu3) ...
Setting up autoconf (2.71-2) ...
Setting up libgcc-11-dev:amd64 (11.3.0-1ubuntu1~22.04) ...
Setting up cpp (4:11.2.0-1ubuntu1) ...
Setting up libc6-dev:amd64 (2.35-0ubuntu3.1) ...
Setting up libtiff5:amd64 (4.3.0-6ubuntu0.3) ...
Setting up libfontconfig1:amd64 (2.13.1-4.2ubuntu5) ...
Setting up libxml2:amd64 (2.9.13+dfsg-1ubuntu0.2) ...
Setting up binutils-x86-64-linux-gnu (2.38-4ubuntu2.1) ...
Setting up automake (1:1.16.5-1.3) ...
update-alternatives: using /usr/bin/automake-1.16 to provide /usr/bin/automake (automake) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/automake.1.gz because associated file /usr/share/man/man1/automake-1.16.1.gz (of link group automake) doesn't exist
update-alternatives: warning: skip creation of /usr/share/man/man1/aclocal.1.gz because associated file /usr/share/man/man1/aclocal-1.16.1.gz (of link group automake) doesn't exist
Setting up binutils (2.38-4ubuntu2.1) ...
Setting up libltdl-dev:amd64 (2.4.6-15build2) ...
Setting up shared-mime-info (2.1-2) ...
Setting up libgd3:amd64 (2.3.0-2ubuntu2) ...
Setting up libstdc++-11-dev:amd64 (11.3.0-1ubuntu1~22.04) ...
Setting up gcc-11 (11.3.0-1ubuntu1~22.04) ...
Setting up libc-devtools (2.35-0ubuntu3.1) ...
Setting up libtool (2.4.6-15build2) ...
Setting up g++-11 (11.3.0-1ubuntu1~22.04) ...
Setting up gcc (4:11.2.0-1ubuntu1) ...
Setting up libgd-perl (2.76-2build1) ...
Setting up g++ (4:11.2.0-1ubuntu1) ...
update-alternatives: using /usr/bin/g++ to provide /usr/bin/c++ (c++) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/c++.1.gz because associated file /usr/share/man/man1/g++.1.gz (of link group c++) doesn't exist
Setting up lcov (1.15-1) ...
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...
root@1d504fa4b33b:/prime_server# less README
root@1d504fa4b33b:/prime_server# apt-get install libcurl4-openssl-dev libzmq3-dev libczmq-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  comerr-dev icu-devtools krb5-multidev libbsd-dev libcurl4 libczmq4 libgssrpc4 libicu-dev libkadm5clnt-mit12 libkadm5srv-mit12
  libkdb5-10 libkrb5-dev libmd-dev libnorm-dev libnorm1 libnspr4 libnspr4-dev libnss3 libnss3-dev libpgm-5.3-0 libpgm-dev
  libsodium-dev libsodium23 libsqlite3-0 libsystemd-dev libxml2-dev libzmq5 uuid-dev
Suggested packages:
  doc-base krb5-doc libcurl4-doc libidn11-dev libldap2-dev librtmp-dev libssh2-1-dev libssl-dev zlib1g-dev krb5-user icu-doc
  libnorm-doc
The following NEW packages will be installed:
  comerr-dev icu-devtools krb5-multidev libbsd-dev libcurl4 libcurl4-openssl-dev libczmq-dev libczmq4 libgssrpc4 libicu-dev
  libkadm5clnt-mit12 libkadm5srv-mit12 libkdb5-10 libkrb5-dev libmd-dev libnorm-dev libnorm1 libnspr4 libnspr4-dev libnss3 libnss3-dev
  libpgm-5.3-0 libpgm-dev libsodium-dev libsodium23 libsqlite3-0 libsystemd-dev libxml2-dev libzmq3-dev libzmq5 uuid-dev
0 upgraded, 31 newly installed, 0 to remove and 0 not upgraded.
Need to get 19.4 MB of archives.
After this operation, 77.4 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libsqlite3-0 amd64 3.37.2-2ubuntu0.1 [641 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 icu-devtools amd64 70.1-2 [197 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy/main amd64 libgssrpc4 amd64 1.19.2-2 [58.4 kB]
Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 libkdb5-10 amd64 1.19.2-2 [40.5 kB]
Get:5 http://archive.ubuntu.com/ubuntu jammy/main amd64 libkadm5srv-mit12 amd64 1.19.2-2 [54.8 kB]
Get:6 http://archive.ubuntu.com/ubuntu jammy/main amd64 libkadm5clnt-mit12 amd64 1.19.2-2 [41.8 kB]
Get:7 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 comerr-dev amd64 2.1-1.46.5-2ubuntu1.1 [41.0 kB]
Get:8 http://archive.ubuntu.com/ubuntu jammy/main amd64 krb5-multidev amd64 1.19.2-2 [125 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy/main amd64 libmd-dev amd64 1.0.4-1build1 [40.0 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy/main amd64 libbsd-dev amd64 0.11.5-1 [168 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libcurl4 amd64 7.81.0-1ubuntu1.7 [289 kB]
Get:12 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libcurl4-openssl-dev amd64 7.81.0-1ubuntu1.7 [385 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnspr4 amd64 2:4.32-3build1 [119 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libnss3 amd64 2:3.68.2-0ubuntu1.1 [1280 kB]
Get:15 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnspr4-dev amd64 2:4.32-3build1 [218 kB]
Get:16 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libnss3-dev amd64 2:3.68.2-0ubuntu1.1 [242 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libnorm1 amd64 1.5.9+dfsg-2 [221 kB]
Get:18 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libpgm-5.3-0 amd64 5.3.128~dfsg-2 [161 kB]
Get:19 http://archive.ubuntu.com/ubuntu jammy/main amd64 libsodium23 amd64 1.0.18-1build2 [164 kB]
Get:20 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libzmq5 amd64 4.3.4-2 [256 kB]
Get:21 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libpgm-dev amd64 5.3.128~dfsg-2 [193 kB]
Get:22 http://archive.ubuntu.com/ubuntu jammy/main amd64 libsodium-dev amd64 1.0.18-1build2 [185 kB]
Get:23 http://archive.ubuntu.com/ubuntu jammy/main amd64 libicu-dev amd64 70.1-2 [11.6 MB]
Get:24 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libxml2-dev amd64 2.9.13+dfsg-1ubuntu0.2 [804 kB]
Get:25 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libnorm-dev amd64 1.5.9+dfsg-2 [389 kB]
Get:26 http://archive.ubuntu.com/ubuntu jammy/main amd64 libkrb5-dev amd64 1.19.2-2 [12.0 kB]
Get:27 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libzmq3-dev amd64 4.3.4-2 [530 kB]
Get:28 http://archive.ubuntu.com/ubuntu jammy/main amd64 uuid-dev amd64 2.37.2-4ubuntu3 [33.1 kB]
Get:29 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libsystemd-dev amd64 249.11-0ubuntu3.6 [306 kB]
Get:30 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libczmq4 amd64 4.2.1-1 [156 kB]
Get:31 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libczmq-dev amd64 4.2.1-1 [399 kB]
Fetched 19.4 MB in 4s (4657 kB/s) 
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libsqlite3-0:amd64.
(Reading database ... 15338 files and directories currently installed.)
Preparing to unpack .../00-libsqlite3-0_3.37.2-2ubuntu0.1_amd64.deb ...
Unpacking libsqlite3-0:amd64 (3.37.2-2ubuntu0.1) ...
Selecting previously unselected package icu-devtools.
Preparing to unpack .../01-icu-devtools_70.1-2_amd64.deb ...
Unpacking icu-devtools (70.1-2) ...
Selecting previously unselected package libgssrpc4:amd64.
Preparing to unpack .../02-libgssrpc4_1.19.2-2_amd64.deb ...
Unpacking libgssrpc4:amd64 (1.19.2-2) ...
Selecting previously unselected package libkdb5-10:amd64.
Preparing to unpack .../03-libkdb5-10_1.19.2-2_amd64.deb ...
Unpacking libkdb5-10:amd64 (1.19.2-2) ...
Selecting previously unselected package libkadm5srv-mit12:amd64.
Preparing to unpack .../04-libkadm5srv-mit12_1.19.2-2_amd64.deb ...
Unpacking libkadm5srv-mit12:amd64 (1.19.2-2) ...
Selecting previously unselected package libkadm5clnt-mit12:amd64.
Preparing to unpack .../05-libkadm5clnt-mit12_1.19.2-2_amd64.deb ...
Unpacking libkadm5clnt-mit12:amd64 (1.19.2-2) ...
Selecting previously unselected package comerr-dev:amd64.
Preparing to unpack .../06-comerr-dev_2.1-1.46.5-2ubuntu1.1_amd64.deb ...
Unpacking comerr-dev:amd64 (2.1-1.46.5-2ubuntu1.1) ...
Selecting previously unselected package krb5-multidev:amd64.
Preparing to unpack .../07-krb5-multidev_1.19.2-2_amd64.deb ...
Unpacking krb5-multidev:amd64 (1.19.2-2) ...
Selecting previously unselected package libmd-dev:amd64.
Preparing to unpack .../08-libmd-dev_1.0.4-1build1_amd64.deb ...
Unpacking libmd-dev:amd64 (1.0.4-1build1) ...
Selecting previously unselected package libbsd-dev:amd64.
Preparing to unpack .../09-libbsd-dev_0.11.5-1_amd64.deb ...
Unpacking libbsd-dev:amd64 (0.11.5-1) ...
Selecting previously unselected package libcurl4:amd64.
Preparing to unpack .../10-libcurl4_7.81.0-1ubuntu1.7_amd64.deb ...
Unpacking libcurl4:amd64 (7.81.0-1ubuntu1.7) ...
Selecting previously unselected package libcurl4-openssl-dev:amd64.
Preparing to unpack .../11-libcurl4-openssl-dev_7.81.0-1ubuntu1.7_amd64.deb ...
Unpacking libcurl4-openssl-dev:amd64 (7.81.0-1ubuntu1.7) ...
Selecting previously unselected package libnspr4:amd64.
Preparing to unpack .../12-libnspr4_2%3a4.32-3build1_amd64.deb ...
Unpacking libnspr4:amd64 (2:4.32-3build1) ...
Selecting previously unselected package libnss3:amd64.
Preparing to unpack .../13-libnss3_2%3a3.68.2-0ubuntu1.1_amd64.deb ...
Unpacking libnss3:amd64 (2:3.68.2-0ubuntu1.1) ...
Selecting previously unselected package libnspr4-dev.
Preparing to unpack .../14-libnspr4-dev_2%3a4.32-3build1_amd64.deb ...
Unpacking libnspr4-dev (2:4.32-3build1) ...
Selecting previously unselected package libnss3-dev:amd64.
Preparing to unpack .../15-libnss3-dev_2%3a3.68.2-0ubuntu1.1_amd64.deb ...
Unpacking libnss3-dev:amd64 (2:3.68.2-0ubuntu1.1) ...
Selecting previously unselected package libnorm1:amd64.
Preparing to unpack .../16-libnorm1_1.5.9+dfsg-2_amd64.deb ...
Unpacking libnorm1:amd64 (1.5.9+dfsg-2) ...
Selecting previously unselected package libpgm-5.3-0:amd64.
Preparing to unpack .../17-libpgm-5.3-0_5.3.128~dfsg-2_amd64.deb ...
Unpacking libpgm-5.3-0:amd64 (5.3.128~dfsg-2) ...
Selecting previously unselected package libsodium23:amd64.
Preparing to unpack .../18-libsodium23_1.0.18-1build2_amd64.deb ...
Unpacking libsodium23:amd64 (1.0.18-1build2) ...
Selecting previously unselected package libzmq5:amd64.
Preparing to unpack .../19-libzmq5_4.3.4-2_amd64.deb ...
Unpacking libzmq5:amd64 (4.3.4-2) ...
Selecting previously unselected package libpgm-dev:amd64.
Preparing to unpack .../20-libpgm-dev_5.3.128~dfsg-2_amd64.deb ...
Unpacking libpgm-dev:amd64 (5.3.128~dfsg-2) ...
Selecting previously unselected package libsodium-dev:amd64.
Preparing to unpack .../21-libsodium-dev_1.0.18-1build2_amd64.deb ...
Unpacking libsodium-dev:amd64 (1.0.18-1build2) ...
Selecting previously unselected package libicu-dev:amd64.
Preparing to unpack .../22-libicu-dev_70.1-2_amd64.deb ...
Unpacking libicu-dev:amd64 (70.1-2) ...
Selecting previously unselected package libxml2-dev:amd64.
Preparing to unpack .../23-libxml2-dev_2.9.13+dfsg-1ubuntu0.2_amd64.deb ...
Unpacking libxml2-dev:amd64 (2.9.13+dfsg-1ubuntu0.2) ...
Selecting previously unselected package libnorm-dev:amd64.
Preparing to unpack .../24-libnorm-dev_1.5.9+dfsg-2_amd64.deb ...
Unpacking libnorm-dev:amd64 (1.5.9+dfsg-2) ...
Selecting previously unselected package libkrb5-dev:amd64.
Preparing to unpack .../25-libkrb5-dev_1.19.2-2_amd64.deb ...
Unpacking libkrb5-dev:amd64 (1.19.2-2) ...
Selecting previously unselected package libzmq3-dev:amd64.
Preparing to unpack .../26-libzmq3-dev_4.3.4-2_amd64.deb ...
Unpacking libzmq3-dev:amd64 (4.3.4-2) ...
Selecting previously unselected package uuid-dev:amd64.
Preparing to unpack .../27-uuid-dev_2.37.2-4ubuntu3_amd64.deb ...
Unpacking uuid-dev:amd64 (2.37.2-4ubuntu3) ...
Selecting previously unselected package libsystemd-dev:amd64.
Preparing to unpack .../28-libsystemd-dev_249.11-0ubuntu3.6_amd64.deb ...
Unpacking libsystemd-dev:amd64 (249.11-0ubuntu3.6) ...
Selecting previously unselected package libczmq4:amd64.
Preparing to unpack .../29-libczmq4_4.2.1-1_amd64.deb ...
Unpacking libczmq4:amd64 (4.2.1-1) ...
Selecting previously unselected package libczmq-dev:amd64.
Preparing to unpack .../30-libczmq-dev_4.2.1-1_amd64.deb ...
Unpacking libczmq-dev:amd64 (4.2.1-1) ...
Setting up libsodium23:amd64 (1.0.18-1build2) ...
Setting up libpgm-5.3-0:amd64 (5.3.128~dfsg-2) ...
Setting up libsqlite3-0:amd64 (3.37.2-2ubuntu0.1) ...
Setting up libnorm1:amd64 (1.5.9+dfsg-2) ...
Setting up libgssrpc4:amd64 (1.19.2-2) ...
Setting up libnspr4:amd64 (2:4.32-3build1) ...
Setting up libpgm-dev:amd64 (5.3.128~dfsg-2) ...
Setting up uuid-dev:amd64 (2.37.2-4ubuntu3) ...
Setting up comerr-dev:amd64 (2.1-1.46.5-2ubuntu1.1) ...
Setting up icu-devtools (70.1-2) ...
Setting up libsodium-dev:amd64 (1.0.18-1build2) ...
Setting up libmd-dev:amd64 (1.0.4-1build1) ...
Setting up libcurl4:amd64 (7.81.0-1ubuntu1.7) ...
Setting up libicu-dev:amd64 (70.1-2) ...
Setting up libbsd-dev:amd64 (0.11.5-1) ...
Setting up libsystemd-dev:amd64 (249.11-0ubuntu3.6) ...
Setting up libkadm5clnt-mit12:amd64 (1.19.2-2) ...
Setting up libnspr4-dev (2:4.32-3build1) ...
Setting up libkdb5-10:amd64 (1.19.2-2) ...
Setting up libzmq5:amd64 (4.3.4-2) ...
Setting up libnss3:amd64 (2:3.68.2-0ubuntu1.1) ...
Setting up libxml2-dev:amd64 (2.9.13+dfsg-1ubuntu0.2) ...
Setting up libcurl4-openssl-dev:amd64 (7.81.0-1ubuntu1.7) ...
Setting up libnss3-dev:amd64 (2:3.68.2-0ubuntu1.1) ...
Setting up libkadm5srv-mit12:amd64 (1.19.2-2) ...
Setting up krb5-multidev:amd64 (1.19.2-2) ...
Setting up libczmq4:amd64 (4.2.1-1) ...
Setting up libnorm-dev:amd64 (1.5.9+dfsg-2) ...
Setting up libkrb5-dev:amd64 (1.19.2-2) ...
Setting up libzmq3-dev:amd64 (4.3.4-2) ...
Setting up libczmq-dev:amd64 (4.2.1-1) ...
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...
root@1d504fa4b33b:/prime_server# ./autogen.sh 
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
configure.ac:8: installing './ar-lib'
configure.ac:8: installing './compile'
configure.ac:9: installing './config.guess'
configure.ac:9: installing './config.sub'
configure.ac:7: installing './install-sh'
configure.ac:7: installing './missing'
Makefile.am: installing './INSTALL'
Makefile.am: installing './depcomp'
parallel-tests: installing './test-driver'
root@1d504fa4b33b:/prime_server# ./configure
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports the include directive... yes (GNU style)
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for ar... ar
checking the archiver (ar) interface... ar
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking whether make supports nested variables... (cached) yes
checking for g++... g++
checking whether the compiler supports GNU C++... yes
checking whether g++ accepts -g... yes
checking for g++ option to enable C++11 features... none needed
checking dependency style of g++... gcc3
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether make sets $(MAKE)... (cached) yes
checking for egrep... (cached) /usr/bin/grep -E
checking whether g++ supports C++11 features by default... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for libzmq >= 4.1.4 libczmq >= 3.0 libcurl >= 7.22.0... yes
checking how to run the C preprocessor... gcc -E
checking whether gcc is Clang... no
checking whether pthreads work with -pthread... yes
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking whether more special flags are required for pthreads... no
checking for PTHREAD_PRIO_INHERIT... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating libprime_server.pc
config.status: creating prime_server/config.h
config.status: executing depfiles commands
config.status: executing libtool commands
root@1d504fa4b33b:/prime_server# git submodule update --init --recursive
Submodule 'logging' (https://gist.github.com/39f2e39273c625d96790.git) registered for path 'src/logging'
Submodule 'testing' (https://gist.github.com/ada902fb51a1ad1e5a27.git) registered for path 'test/testing'
Cloning into '/prime_server/src/logging'...
Cloning into '/prime_server/test/testing'...
Submodule path 'src/logging': checked out '10ba199d1e23b356c3446b723289a602663d41ec'
Submodule path 'test/testing': checked out '1c45352fa2c4e25a748cf1221b93a9aed4dbc34c'
root@1d504fa4b33b:/prime_server# make test -j
make test -j
  CXX      src/prime_serverd-prime_serverd.o
  CXX      src/libprime_server_la-prime_server.lo
  CXX      src/libprime_server_la-netstring_protocol.lo
  CXX      src/libprime_server_la-http_protocol.lo
  CXX      src/libprime_server_la-http_util.lo
  CXX      src/prime_httpd-prime_httpd.o
  CXX      src/prime_proxyd-prime_proxyd.o
  CXX      src/prime_workerd-prime_workerd.o
  CXX      src/prime_echod-prime_echod.o
  CXX      src/prime_filed-prime_filed.o
  CXXLD    libprime_server.la
  CXXLD    prime_httpd
  CXXLD    prime_serverd
  CXXLD    prime_proxyd
  CXXLD    prime_workerd
  CXXLD    prime_filed
  CXXLD    prime_echod
make  test/zmq test/netstring test/http test/shaping test/interrupt
make[1]: Entering directory '/prime_server'
  CXX      test/zmq-zmq.o
  CXX      test/netstring-netstring.o
  CXX      test/http-http.o
  CXX      test/shaping-shaping.o
  CXX      test/interrupt-interrupt.o
  CXXLD    test/zmq
  CXXLD    test/interrupt
  CXXLD    test/shaping
  CXXLD    test/netstring
  CXXLD    test/http
make[1]: Leaving directory '/prime_server'
make  check-TESTS
make[1]: Entering directory '/prime_server'
make[2]: Entering directory '/prime_server'
PASS: test/zmq
PASS: test/shaping
PASS: test/interrupt
PASS: test/netstring
PASS: test/http
============================================================================
Testsuite summary for prime_server 0.7.0
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[2]: Leaving directory '/prime_server'
make[1]: Leaving directory '/prime_server'
```